### ISSUE TITLE
Be able to change a Collection name

### ISSUE THREAD

**Issue by asitemade4u** [directus/api#1812]
# Feature Request
Sometimes, a Collection begins small and ends up aggregating several tables in the MySQL database.
It is then necessary to change its name.
By fumbling within the `directus` MySQL database tables I was able to change the name of one of my collections quite easily.

## What problem does this feature solve?
Changes in a Collection span or use

## How do you think this should be implemented?
- Either by allowing the change from the interface
- or by giving the developers a SQL script

**Comment by benhaynes** [comment]
Some thoughts:

You can do this now, on the surface, using Collection Translations (in Settings). This will update the name in the App for all users.

To update the actual name is more complex, and also breaks (_changes_) the project's API... so it should be used with extreme caution. It's mostly a time/resourcing thing (oss budget), but it would be a lot of work to do this right since there are SO many places that use the Collection name (eg: `directus_fields`, `directus_activity`, `directus_collection_presets`, `directus_collections`, etc.. almost all system tables). Also, it's very possible that User data references these names too... and we have a strict policy on not updating/changing user data.

So I think your idea of a script in the Docs might be best... that adds just enough barrier-to-entry so that devs know this is serious, but makes changing Collection names (which sometimes you _know_ won't break anything) a lot easier.

**Comment by rijkvanzanten** [comment]
That being said, from a "directus as db tool" perspective it is something that you'd expect to be possible

**Comment by benhaynes** [comment]
True! We could do this, but let users know we won't be updating THEIR content... and cite our policy on that.

**Comment by piotr-cz** [comment]
Would be nice to have a walkthrough guide to change collection key in docs.
When prototyping new project/ creating POC it's quite common to change schema
